packages/ui/src/app/GlobalModals.tsx

```diff
@@ -217,6 +222,9 @@ export const GlobalModals = () => {
 
   const potentialFallback = useGlobalModalHandler(currentModalMachine, hideModal)
 
+  if (!supportTransactions && !NON_TRANSACTIONAL_MODALS.includes(modal as ModalNames))
+    return <NotSupportMobileModal onClose={hideModal} />
+
   if (modal && !GUEST_ACCESSIBLE_MODALS.includes(modal as ModalNames) && !activeMember) {
```

Comment:
```suggestion
  if (!modal) {
    return null
  }

  if (!supportTransactions && !NON_TRANSACTIONAL_MODALS.includes(modal as ModalNames)) {
    return <NotSupportMobileModal onClose={hideModal} />
  }

  if (!GUEST_ACCESSIBLE_MODALS.includes(modal as ModalNames) && !activeMember) {
```

Reply:
Please remember to test your code on mobile before pushing it

Reply:
Thanks for your carefully review